magento/magento2
# Added acl resource to Module Marketlace

### Fixed Issues (if relevant)

1. magento/magento2#15632: No ACL set for marketplace

### Manual testing scenarios
1. Create admin role.
2. Set custom role resources.
3. Select all resources.
4. Add a user to the role.
5. Login as this user.
6. Search for ' Marketplace'.

Expected result:
Marketplace is present on the resources list.
Actual result:
Marketplace is not present on the resources list.

### Contribution checklist
